Wishlist/wishlist.html

```diff
@@ -0,0 +1,76 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://airave-ui.netlify.app/component.css" rel="stylesheet">
+    <title>Airave Mart</title>
+</head>
+<body>
+    <div class="nav">
+        <div class="nav-brand ml-5">
+            <div class="nav-logo">Logo</div> 
+                <a href="#"><p class="brand">Airave Mart</p></a>
+            </div>
+            <div class="searchbox mr-2">
+                <input type="text" class="search-input">
+                 <i class="bi bi-search search-icon"></i>
+             </div>
+        <ul class="mr-5 gap-1">
+            <li>
+                <a href="/Cart/cart.html"><div class="badge-wrapper">
+                <i class="bi bi-cart fa-lg"></i>
+                <span class="icon-badge flex-align-center">2</span>
+                </div></a>
+            </li>
+            <li><a href="/Wishlist/wishlist.html"><i class="bi bi-heart"></i></a></li>
+            <li><a href="#"><i class="bi bi-person-fill"></i></a></li>
+        </ul>
+    </div> 
+    <h4 class="flex-align-center mt-2 mb-2">My Wishlist</h4>
+
+    <div class="flex gap-3 ml-5 mr-5">
+        <div class="card box-shadow" id="card-block">
+            <i class="fa fa-times dismiss"></i>
+            <img src="/assest/adidas.jpg" class="card-img" alt="card-image"/>
+            <div class="card-body">
+                <div class="card-title">American Tourister</div>
+                <div class="card-description">This is a new quality bag.</div>
+                <div class="price">Rs. 1234</div>
+                <a href="/Cart/cart.html"><button class="btn btn-accent width-100 mt-1">Add to Cart</button></a>
+            </div>  
+        </div>
+        <div class="card box-shadow" id="card-block">
+            <i class="fa fa-times dismiss"></i>
+            <img src="/assest/adidas.jpg" class="card-img" alt="card-image"/>
+            <div class="card-body">
+                <div class="card-title">American Tourister</div>
+                <div class="card-description">This is a new quality bag.</div>
+                <div class="price">Rs. 1234</div>
+                <a href="/Cart/cart.html"><button class="btn btn-accent width-100 mt-1">Add to Cart</button></a>
+            </div>  
+        </div>
+        <div class="card box-shadow" id="card-block">
+            <i class="fa fa-times dismiss"></i>
+            <img src="/assest/adidas.jpg" class="card-img" alt="card-image"/>
+            <div class="card-body">
+                <div class="card-title">American Tourister</div>
+                <div class="card-description">This is a new quality bag.</div>
+                <div class="price">Rs. 1234</div>
+                <a href="/Cart/cart.html"><button class="btn btn-accent width-100 mt-1">Add to Cart</button></a>
+            </div>  
+        </div>
+        <div class="card box-shadow" id="card-block">
```

Comment:
The same id has been used twice, they are supposed to be unique to one element. Please check.

Reply:
Changed it